### PR TITLE
update death wish & sweeping strike templates for bcc, fixes #3160

### LINKED ISSUE #3160: new from template warrior death wish talent show sweeping strike
### Description

expected death wish talent as i chose, it loaded sweeping strike instead

### WeakAuras Version

3.4.2

### World of Warcraft Flavor

The Burning Crusade

### Tested with only WeakAuras

- [ ] Yes
- [X] No

### Lua Error

_No response_

### Reproduction Steps

New Aura
From Template
Icon 
Abilities
Death Wish

### Last Good Version

_No response_

### Screenshots

_No response_

### Export String

_No response_

### PR DESCRIPTION
**Pull request by mrbuds**
fixes #3160

### CODE REVIEW

File: WeakAurasTemplates/TriggerTemplatesDataBCC.lua [hunk 1]
```diff
@@ -110,11 +110,11 @@ templates.class.WARRIOR = {
         { spell = 7384, type = "ability", requiresTarget = true, form = 1}, -- Overpower
         { spell = 7386, type = "ability", requiresTarget = true, debuff = true}, -- Sunder Armor
         { spell = 12323, type = "ability", debuff = true, talent = 46}, -- Piercing Howl
-        { spell = 12292, type = "ability", buff = true, talent = 53}, -- Sweeping Strikes
+        { spell = 12328, type = "ability", buff = true, talent = 53}, -- Sweeping Strikes
```
Comment (InfusOnWoW) [inline WeakAurasTemplates/TriggerTemplatesDataBCC.lua:113]: Eh the old thing looks correct to me:
13292 is sweeping strikes, and it's talent number 1, 13
123228 is death wish and it's talent number 0, 13

What is going on?

Comment (mrbuds) [reply WeakAurasTemplates/TriggerTemplatesDataBCC.lua:113]: Those 2 talents were inverted between Arms branch & Fury Branch on BCC
image
image

Comment (InfusOnWoW) [reply WeakAurasTemplates/TriggerTemplatesDataBCC.lua:113]: I don't understand what is currently wrong:

Death Wish, spell id: 12328 https://classic.wowhead.com/spell=12328/death-wish is talent 0 + 13 = 13
Sweeping Strikes, spell id:  12292 https://classic.wowhead.com/spell=12292/sweeping-strikes is talent 1 +13 = 53

The switch is in the BCC data compared to vanilla classic.

Comment (mrbuds) [reply WeakAurasTemplates/TriggerTemplatesDataBCC.lua:113]: <https://tbc.wowhead.com/spell=12328/>
<https://tbc.wowhead.com/spell=12292>

Comment (InfusOnWoW) [reply WeakAurasTemplates/TriggerTemplatesDataBCC.lua:113]: Oh wow.

**Pull request merged**